src/sutta_publisher/shared/config.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from typing import cast
+
+import inject
+
+from sutta_publisher.publishers.base import ActivePublishers
+from sutta_publisher.publishers.html import HtmlPublisher
+
+
+@dataclass(frozen=True)
+class Config:
+    bilara_data_url = "https://github.com/suttacentral/bilara-data"
+    publication_number: str
+
+    @classmethod
+    def from_publication(cls, publication_number: str) -> Config:
+        return cls(publication_number=publication_number)
+
+
+def setup_logging() -> None:
+    log_format = "[%(levelname)7s] %(filename)s: %(message)s"
+    logging.basicConfig(level=logging.INFO, format=log_format, datefmt="%Y-%m-%d %H:%M:%S")
+
+
+def setup_inject(publication_number: str, bindings: dict = None) -> None:
+    """Setup configuration for the application."""
```

Comment:
@janwadolowski propose text what would help in understanding that more

Reply:
I read a bit on the topic of dependencies injection. To be honest I have no idea how to summarize this in a short comment for someone who doesn't understand the topic. Whereas if someone understands the concept there is no need to explain this I guess?

Reply:
ok, if you will have an idea about the explanation comment just add it later. You can treat it here as a 'global' lazy dict which maps key's to objects (here classes to concrete instances)